system/make.py

```diff
@@ -90,9 +93,9 @@ def sanitize_output(output):
     :return: sanitized output
     """
     if output is None:
-        return b('')
+        return ''
```

Comment:
Yep, now that module.run_command() returns native strings this looks like the correct behaviour.